src/fosslight_reuse/_fosslight_reuse.py

```diff
@@ -254,6 +257,8 @@ def reuse_for_project(path_to_find):
         print_error(f"Error_Reuse_lint: {ex}")
         error_occured = True
 
+    logger.warning(f"missing_license : {missing_license}")
```

Comment:
Why print this here?

Reply:
It was just printing msg for debugging.
I deleted it.